SKILL.tsv

```diff
@@ -4803,7 +4803,7 @@ SKILL_20150717_004802	Physical ATK and Magic ATK will be increased for 30 mins f
 SKILL_20150717_004803	$Fulldraw: Knockback Increase
 SKILL_20150717_004804	{memo X}$When equipped with a [Two-handed Bow], adds 20% additional physical attack damage against airborne monsters.
 SKILL_20150717_004805	$While [Swift Step] is active, enemies that are hit by [Oblique Shot] will be affected by [Slow] for 7 seconds with a chance of 5% per attribute level. 
-SKILL_20150717_004806	Physical DEF of the enemies that are threaded with [Full Draw] will decrease by 10% per Attribute level.
+SKILL_20150717_004806	Physical DEF of the enemies that are threaded with [Fulldraw] will decrease by 10% per Attribute level.
```

Comment:
$Decreases the physical defense of enemy that are threaded with [Fulldraw] by 10% per attribute level.